resources/views/pages-legacy/controlpanel.blade.php

```diff
@@ -102,12 +109,178 @@ function (data) {
+        success: () => {
+            showStatusSuccess('{{ __("legacy.success.change") }}');
+        }
+    });
+}
+
+function handleResetWebApiKeyClick() {
+    if (!confirm('Are you sure you want to reset your web API key?')) {
+        return;
+    }
+
+    $.ajax({
+        url: '{{ route('settings.keys.web.destroy') }}',
+        type: 'DELETE',
+        success: () => {
+            showStatusSuccess('{{ __("legacy.success.reset") }}');
+
+            // Temporary, will be removed in the Inertia migration.
+            setTimeout(() => {
+                window.location.reload();
+            }, 1000)
+        }
+    });
+}
+
+function handleResetConnectApiKeyClick() {
+    if (!confirm('Are you sure you want to reset your Connect API key?')) {
+        return;
+    }
+
+    $.ajax({
+        url: '{{ route('settings.keys.connect.destroy') }}',
+        type: 'DELETE',
+        success: () => {
+            showStatusSuccess('{{ _("legacy.success.reset") }}');
+        }
+    });
+}
+
+function handleDeleteAllUserComments() {
+    if (!confirm('Are you sure you want to permanently delete all comment on your wall?')) {
+        return;
+    }
+
+    $.ajax({
+        url: '{{ route('user.comment.destroyAll', $userModel) }}',
+        type: 'DELETE',
+        success: () => {
+            showStatusSuccess('{{ __("legacy.success.delete") }}');
+        }
+    });
+}
+
+function handleChangePasswordSubmit(formValues) {
+    const { currentPassword, newPassword, confirmPassword } = formValues;
+
+    if (newPassword !== confirmPassword) {
+        alert("Make sure the New Password and Confirm Password values match.");
+        
+        return;
+    }
+
+    $.ajax({
+        url: '{{ route('settings.password.update') }}',
+        type: 'PUT',
+        contentType: 'application/json; charset=utf-8',
+        dataType: 'json',
+        data: JSON.stringify({ currentPassword, newPassword }),
+        success: () => {
+            showStatusSuccess('{{ __("legacy.success.change") }}');
+        }
+    });
+}
+
+function handleChangeEmailSubmit(formValues) {
+    const { newEmail, confirmEmail } = formValues;
+
+    if (newEmail !== confirmEmail) {
+        alert("Make sure the New Email Address and Confirm Email Address values match.");
+        
+        return;
+    }
+
+    const hasDevPermissions = {{ $permissions >= Permissions::Developer ? 'true' : 'false' }};
+    if (
+        hasDevPermissions
+        && !confirm('Changing your email address will revoke your privileges and you will need to have them restored by staff. Are you sure you want to continue?')
+    ) {
+        return;
+    }
+
+    $.ajax({
+        url: '{{ route('settings.email.update') }}',
+        type: 'PUT',
+        contentType: 'application/json; charset=utf-8',
+        dataType: 'json',
+        data: JSON.stringify({ newEmail }),
+        success: () => {
+            showStatusSuccess('{{ __("legacy.success.change") }}')
+        }
```

Comment:
My email was successfully updated, but an error was still generated:
![image](https://github.com/user-attachments/assets/fa963479-ac2f-4a6b-a898-b4e4cba920b4)

No email was sent.

Reply:
A different error is generated when trying to manually resend the verification email.
![image](https://github.com/user-attachments/assets/47b395e2-f504-400b-971d-80ace0234b97)

Reply:
Both issues should now be fixed in latest.